tests/figures/chart/charts_zoom.test.ts

```diff
@@ -0,0 +1,454 @@
+    ).toBeTruthy();
+    expect(fixture.querySelector(".o-master-chart-container")).not.toBeNull();
+  });
+
+  test("Disabling zoom removes the master chart", async () => {
+    await mountSpreadsheet();
+    createTestChart(chartId, {}, { zoomable: true });
+    await openChartDesignSidePanel(model, env, fixture, chartId);
+
+    expect(
+      (model.getters.getChartDefinition(chartId) as LineChartDefinition).zoomable
+    ).toBeDefined();
+
+    await simulateClick("input[name='zoomable']");
+    expect((model.getters.getChartDefinition(chartId) as LineChartDefinition).zoomable).toBeFalsy();
+    expect(fixture.querySelector(".o-master-chart-container")).toBeNull();
+  });
+
+  describe("line chart", () => {
+    beforeEach(async () => {
+      await mountSpreadsheet();
+      createTestChart(chartId, {}, { type: "line", zoomable: true });
+      await nextTick();
+    });
+
+    test("Can select on the master chart", async () => {
+      const element = fixture.querySelector(".o-master-chart-canvas") as HTMLCanvasElement;
+      const { left, top, width, height } = element.getBoundingClientRect();
+      const startX = left + width / 3 - 5;
+      const startY = top + height / 2;
+      const offsetX = width / 3 + 10;
+      triggerMouseEvent(element, "pointerdown", startX, startY);
+      triggerMouseEvent(element, "pointermove", startX + offsetX, startY);
+      triggerMouseEvent(element, "pointerup", startX + offsetX, startY);
+      await nextTick();
+      const store = env.getStore(ZoomableChartStore);
+      const { min: newMin, max: newMax } = store.currentAxesLimits[lineChartId]?.x ?? {};
+      expect(newMin).toEqual(1);
+      expect(newMax).toEqual(2);
+    });
+
+    test("Can select from the left bound on the master chart", async () => {
+      const element = fixture.querySelector(".o-master-chart-canvas") as HTMLCanvasElement;
+      const { left, top, width, height } = element.getBoundingClientRect();
+      const startX = left;
+      const startY = top + height / 2;
+      const offsetX = width / 2;
+      triggerMouseEvent(element, "pointerdown", startX, startY);
+      triggerMouseEvent(element, "pointermove", startX + offsetX, startY);
+      triggerMouseEvent(element, "pointerup", startX + offsetX, startY);
+      await nextTick();
+      const store = env.getStore(ZoomableChartStore);
+      const { min: newMin, max: newMax } = store.currentAxesLimits[lineChartId]?.x ?? {};
+      expect(newMin).toEqual(2);
+      expect(newMax).toEqual(3);
+    });
+
+    test("Can select from the right bound on the master chart", async () => {
+      const element = fixture.querySelector(".o-master-chart-canvas") as HTMLCanvasElement;
+      const { left, top, width, height } = element.getBoundingClientRect();
+      const startX = left + width;
+      const startY = top + height / 2;
+      const offsetX = -width / 2;
+      triggerMouseEvent(element, "pointerdown", startX, startY);
```

Comment:
these steps can be replaced by the async helper `dragElements`
we could rename it as `MouseDrag` or `ClickAndDrag` as we are not explicitely dragging the element, it's just that's it's always been used like this up until now